pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
```

Comment:
Perhaps this will read better?
```suggestion
- List devices with an available brightness control:
```

Reply:
I don't think it's neseccary.

Reply:
Me neither. Though tt could very well be 
`- List devices with available brightness controls:`
 considering the fact that there could be multiple devices with multiple controls.

Reply:
I think both versions are correct and I don't really prefer any of them.

Reply:
I don't like the usage of available

Maybe this fits better?

```suggestion
- List devices with changeable brightness:
```